DESCRIPTION

```diff
@@ -27,6 +27,8 @@ Imports:
     grDevices,
     lubridate,
     methods,
+    miniUI,
```

Comment:
Ideally this would be optional (suggested) and a check would be done for it when using the functionality, is that possible?  You want to keep the imports to as minimal as necessary and if someone doesn't use the configurator they shouldn't need to install this.  Just a thought to keep the dependencies down, not required.

Reply:
OK. I will check and update with next update. Thank you